### PR TITLE
Add ProjectPlatform.last_seen index

### PR DESCRIPTION
**Pull request by ehfeng**

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR includes migrations</td>
    </tr>
  </tbody>
</table>

## Migration Checklist

- [ ] new columns need to be nullable (unless table is new)
- [ ] migration with any new index needs to be done concurrently
- [ ] data migrations should not be done inside a transaction
- [ ] before merging, check to make sure there aren't conflicting migration ids


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/south_migrations/0344_add_index_ProjectPlatform_last_seen.py [hunk 1]
```diff
@@ -0,0 +1,3245 @@
...
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+    def forwards(self, orm):
+        # Adding index on 'ProjectPlatform', fields ['last_seen']
+        db.create_index('sentry_projectplatform', ['last_seen'])
```
Comment (mattrobenolt) [inline src/sentry/south_migrations/0344_add_index_ProjectPlatform_last_seen.py]: Needs to be created concurrently. Search for `CREATE INDEX CONCURRENTLY` in other migrations.

**Review by mattrobenolt — CHANGES_REQUESTED** [review]

**Comment by mattrobenolt** [comment]
And needs changes.

File: src/sentry/south_migrations/0344_add_index_ProjectPlatform_last_seen.py (hunk 1 above)
Comment (ehfeng) [inline src/sentry/south_migrations/0344_add_index_ProjectPlatform_last_seen.py]: Fixed

**Comment by ehfeng** [comment]
Added changes

File: CHANGES [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Version 8.20 (Unreleased)
 -------------------------
 
+Schema Changes
```
Comment (mattrobenolt) [inline CHANGES]: Once #5853 is merged, this will conflict.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**